src/app/cluster/duck/reducers.ts

```diff
@@ -34,7 +34,16 @@ export const addClusterSuccess = (state = INITIAL_STATE, action) => {
   };
 };
 export const removeClusterSuccess = (state = INITIAL_STATE, action) => {
-  return { ...state };
+  return {
+    ...state,
+    clusterList: state.clusterList.filter(item => item.Cluster.metadata.name !== action.name),
```

Comment:
We been using the MigCluster object (not Cluster) to reference the metadata.name value. This is a nitpick but we should be consistent with how we are accessing this data.

Reply:
Will update.